pointmatcher/TransformationsImpl.h

```diff
@@ -72,6 +73,7 @@ struct TransformationsImpl
 		}
 		
```

Comment:
We were missing `SimilarityTransformation() : Transformation("SimilarityTransformation",  ParametersDoc(), Parameters()) {}`.